app/models/contact.rb

```diff
@@ -1,17 +1,12 @@
 class Contact < ApplicationRecord
-  RELATIONSHIP_TYPES = %w[friend family colleague]
+  RELATIONSHIP_TYPES = %i[friend family colleague enemy].freeze
 
   belongs_to :user
-
   has_many :interactions, dependent: :destroy
 
-  validates :user_id, presence: true
-  validates :name, :relationship,  presence: true
-  validates :name, :relationship, :email, :phone_number, :address, :social_media_handle, presence: true
+  validates :name, :relationship, :user_id, presence: true
```

Comment:
Do we need the `user_id` here since we already have the `belongs_to :user`?

Reply:
Nope, thank you! 🌈